### DISCUSSION TITLE
Chrome keeps adding popup dialogs

### DISCUSSION THREAD

**Discussion by ccea-dev-team** [selenide/selenide#2767] [General]
Had issues in the past with Chrome showing popup dialogs for password security etc knocking off tests i.e. https://github.com/selenide/selenide/discussions/2658

They have recently added new ones for bookmarks and reading view (see below).  These block any clicks in the top corner.

The issue seems to be either there is no option to disable these OR that the options are completely unknown with no documentation.

Also there is no point raising these problems in the chrome driver bug management tool as they never reply  https://bugs.chromium.org/p/chromedriver/issues/list?sort=-id

How do you Selenide guys find out these options and do you know any for disabling the new reading view popup?

![image](https://github.com/selenide/selenide/assets/99395943/8369bf31-a6fd-4943-843d-ca60b74621a8)

**Comment by asolntsev** [comment]
@ccea-dev-team I also don't know where to find the right properties. :(

**Comment by ccea-dev-team** [comment]
The problem has been raised by someone over at the ChromeDriver issue log - https://issues.chromium.org/issues/349952798

I added a comment that it would be good if they provided documentation for ChromeDriver options to disable these kind of popups as it has happened before with password security breach and insecure downloads warnings.
